packages/cli/src/index.ts

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-import dotenv from 'dotenv';
+import 'dotenv/config';
```

Comment:
praise: I like little improvements like these and others in the PR (that I didn't specifically point out)!